oe_bootstrap_theme.theme

```diff
@@ -61,7 +61,15 @@ function oe_bootstrap_theme_theme_suggestions_views_view_table_alter(array &$sug
  * Implements hook_preprocess().
  */
 function oe_bootstrap_theme_preprocess(&$variables) {
-  $variables['bcl_icon_path'] = base_path() . \Drupal::service('extension.list.theme')->getPath('oe_bootstrap_theme') . '/assets/icons/bcl-default-icons.svg';
+  // Get the active theme and its path.
+  $theme = \Drupal::theme()->getActiveTheme();
+  $theme_path = $theme->getPath($theme->getName());
```

Comment:
This method doesn't accept parameters, the test are failing which is good because is telling us something is not ok.

Reply:
done.